gremlin-console/pom.xml

```diff
@@ -39,6 +39,12 @@ limitations under the License.
             <artifactId>tinkergraph-gremlin</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
```

Comment:
Is this necessary? We purposely put jackson in gremlin-shaded to avoid doing things like this.

Reply:
it doesn't work without this

Reply:
what doesn't work?

Reply:
aws sdk used for sigv4 require jackson-databind
```
cluster = Cluster.build("localhost").port(8182).auth(sigv4("us-west2")).create()
g = traversal().withRemote(DriverRemoteConnection.using(cluster, "g"))
```
leads to 
`java.lang.NoClassDefFoundError: com/fasterxml/jackson/databind/ObjectMapper`